content/english/python-basics/data-structures/lists.md

```diff
@@ -106,15 +109,13 @@ If you don't specify the index,`list.pop()` then it removes the last element fro
 `list.reverse()` function reverse the elements of the list.
 
 ```python
-
 fruits.reverse()
 #Now the list is ['lemon', 'mango', 'apple', 'banana', 'guava']
-
 ```
 
 ### Get the minimum and maximum element of the list
 
-`min(list)` function returns the minimum element of the list and the maximum element can be found by using the `max(list)` function.
+`min(list)` function returns the minimum element of the list and the maximum element can be found by using the `max(list)` function. If your list is a list of numbers, it will return the lowest or highest value number of the list. If your list is strings, it will return the first or last alphabetically.
```

Comment:
```suggestion
`min(list)` function returns the minimum element of the list and the maximum element can be found by using the `max(list)` function. If your list is a list of numbers, it will return the lowest or highest value number of the list. If your list is a list of strings, it will return the first or last value alphabetically.
```